integration/test/performance_tracker/performance_query_utils.go

```diff
@@ -0,0 +1,78 @@
+package data_collector
+
+import (
+	"time"
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
+)
+
+const (
+	Namespace = "CWAgent"
+	DimensionName = "InstanceId"
+	Stat = "Average"
+	Period = 30
+)
+
+// CWGetMetricDataAPI defines the interface for the GetMetricData function
+type CWGetMetricDataAPI interface {
+	GetMetricData(ctx context.Context, params *cloudwatch.GetMetricDataInput, optFns ...func(*cloudwatch.Options)) (*cloudwatch.GetMetricDataOutput, error)
+}
+
+// GetMetrics Fetches the cloudwatch metrics for your provided input in the given time-frame
+func GetMetrics(c context.Context, api CWGetMetricDataAPI, input *cloudwatch.GetMetricDataInput) (*cloudwatch.GetMetricDataOutput, error) {
+	return api.GetMetricData(c, input)
+}
```

Comment:
This is a pass-through function that I don't think adds a lot of value. All we do is take in the required inputs to call the `GetMetricData` API, and then return exactly what the client returns. This function is not necessary since we could just call the API via the SDK client directly.